docs/website/blog/3.0.0-GA-blog.md

```diff
@@ -0,0 +1,333 @@
+#### Detect appropriate Devfile
+`odo init` will automatically analyze your directory and based on the project type, suggest an appropriate Devfile for use.
+
+#### Personalize Devfile configuration
+`odo init` also provides a better interactive way to configure your Devfile according to your project requirement. With this feature, you can modify the environment variables exported to and ports exposed by your application on the cluster.
+
+### odo dev
+`odo dev` allows you to code, build, run, debug and test your application on the cluster in a continuous workflow.
+
+```shell
+$ odo dev
+  __
+ /  \__     Developing using the "my-node-app" Devfile
+ \__/  \    Namespace: default
+ /  \__/    odo version: v3.0.0
+ \__/
+
+↪ Deploying to the cluster in developer mode
+ •  Waiting for Kubernetes resources  ...
+ ⚠  Pod is Pending
+ ✓  Pod is Running
+ ✓  Syncing files into the container [172ms]
+ ✓  Building your application in container on cluster (command: install) [11s]
+ •  Executing the application (command: run)  ...
+ -  Forwarding from 127.0.0.1:40001 -> 3000
+
+
+↪ Dev mode
+ Status:
+ Watching for changes in the current directory /tmp/pd
+
+ Keyboard Commands:
+[Ctrl+c] - Exit and delete resources from the cluster
+     [p] - Manually apply local changes to the application on the cluster
+```
+
+_How is it different from `odo push` command of v2?_
+
+odo v3 combines the `odo push` and `odo watch` commands from v2 into a single `odo dev` command.
+
+Additionally it also provides,
+* [Continuous workflow](#continuous-workflow)
+* [Manual trigger to apply local changes on the cluster](#manual-trigger-to-apply-local-changes-on-the-cluster)
+* [Resource cleanup](#resource-cleanup)
+* [Better watch system](#better-watch-system)
+* [Port forwarding to access the application](#port-forwarding-to-access-the-application)
+
+#### Continuous workflow
+`odo dev` is a non-terminating command that runs until user hits `Ctrl+c`. It continuously watches the directory for any new changes (including the changes occuring in the Devfile) and automatically syncs them with the application running on the cluster. 
+
+#### Manual trigger to apply local changes on the cluster
+By default, `odo dev` automatically detects any new change in the directory, and syncs it with the application running on the cluster, but it is possible to do this trigger manually by running the command with a `--no-watch` flag and pressing `p` to trigger the sync.
+
+#### Better watch system
+`odo dev` continuously monitors the resources created to run the application on the cluster and ensures that the application always stays running.
+
+#### Resource cleanup
+Before exiting, `odo dev` will cleanup any resources that it created for running the application on the cluster.
+
+#### Port forwarding to access the application
+`odo dev` uses port-forwarding to allow application access instead of Routes and Ingress that were used in `odo push`, making it much faster to access the application.
```

Comment:
```suggestion
`odo dev` uses port-forwarding to allow application access instead of Routes and Ingress that were used in `odo push`, making it much easier to access the application.
```